client/packages/host/src/api/api.ts

```diff
@@ -20,7 +20,10 @@ export const getHostQueries = (sdk: Sdk) => ({
     },
     syncStatus: async () => {
       const result = await sdk.syncStatus();
-      return result?.latestSyncStatus;
+      return result?.syncStatus;
+    },
+    syncInfo: async () => {
+      return await sdk.syncInfo();
```

Comment:
is `return await` going to be an issue here? Seems redundant (just `syncInfo: sdk.syncInfo` I think?) but what happens if `syncInfo` throws an error? 🤔 Does the promise get rejected or is it a thrown error? (I can't test 😁)

Reply:
just tried, some type differences, and useQuery complains (need more typing there) : / 

![Screen Shot 2022-10-03 at 10 43 00 PM](https://user-images.githubusercontent.com/26194949/193547822-ac9d5f20-e0ec-4b07-adf9-d0639a6b9a63.png)
![Screen Shot 2022-10-03 at 10 42 11 PM](https://user-images.githubusercontent.com/26194949/193547828-9bafe5d4-dd42-4e06-9124-90dc0134ac1f.png)

![Screen Shot 2022-10-03 at 10 48 07 PM](https://user-images.githubusercontent.com/26194949/193548688-e6a44409-7f36-46fc-b984-c5626fd6d323.png)

Reply:
🙏 259ea8210d24cfa7583d1413d9e7ed075800b2ed